### DISCUSSION TITLE
软路由 内部不能科学上网，但是连接软路由的设备可以科学上网，什么原因？

### DISCUSSION THREAD

**Discussion by jrt324** [vernesong/OpenClash#3148] [Q&A]
软路由安装了 openclash ，所有连接到路由器的设备都能科学上网，但是软路由的内部不能科学上网，ssh 到软路由的终端 curl google.com、youtube 等都连接失败， 但是为什么我其他连接到这个路由器的设备都能科学上网呢？各位大神，这是什么原因？

网络结构很简单： 软路由是直接连接光猫的，而其他还有几个 WIFI 路由是链接到这个软路由

**Comment by CNcheukho** [comment]
同样遇到这个问题，在电脑打开openclash的情况下 再打开了clash之后 关掉clash，直接没办法科学上网了。连接方式和楼主一样，其他通过wifi连接的手机设备都可以正常科学上网
